docs/source/using/managing-minishift.adoc

```diff
@@ -293,23 +293,32 @@ CONTAINER    IMAGE                   COMMAND                CREATED        STATU
 71fe8ff16548 openshift/origin:v1.5.1 "/usr/bin/openshift s" 4 minutes ago  Up 4 minutes  origin
 ----
 
-[[experimental-functionality]]
-== Experimental functionality
+[[enabling-experimental-openshift-features]]
+== Enabling Experimental OpenShift Features
```

Comment:
It might not be OpenShift. Right now we use it to expose some OpenShift/cluster up functionality, but I would imagine that we could use it for Minishift experimental features as well.